services/movie_session.py

```diff
@@ -0,0 +1,59 @@
+from datetime import datetime
+
+from django.db.models import QuerySet
+
+from db.models import MovieSession
+
+
+def create_movie_session(movie_show_time: datetime,
+                         movie_id: int,
+                         cinema_hall_id: int) -> MovieSession:
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id
+    )
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet:
+    queryset = MovieSession.objects.all()
+
+    if session_date:
+        year = session_date.split("-")[0]
+        month = session_date.split("-")[1]
+        day = session_date.split("-")[2]
+        queryset = queryset.filter(show_time__year=year,
+                                   show_time__month=month,
+                                   show_time__day=day,)
```

Comment:
try to use filter by `show_time__date` , you can remove these splits

Reply:
Done.